zerver/tests/test_retention.py

```diff
@@ -81,15 +84,19 @@ def _make_expired_zulip_messages(self, message_quantity: int) -> List[int]:
 
         return msg_ids
 
-    def _verify_archive_data(self, expected_message_ids: List[int]) -> None:
+    def _get_usermessage_ids(self, message_ids: List[int]) -> List[int]:
+        return list(UserMessage.objects.filter(message_id__in=message_ids).values_list('id', flat=True))
+
+    def _verify_archive_data(self, expected_message_ids: List[int],
+                             expected_usermessage_ids: List[int]) -> None:
         self.assertEqual(
             set(ArchivedMessage.objects.values_list('id', flat=True)),
-            set(Message.objects.filter(id__in=expected_message_ids).values_list('id', flat=True))
+            set(expected_message_ids)
```

Comment:
Haha, the old code here was just silly, wasn't it?

Reply:
Yeah, I don't know how/why I missed this part when doing a general rewrite of a lot of the tests earlier :see_no_evil: